static/templates/user_info_popover_content.handlebars

```diff
@@ -32,7 +32,17 @@
         <li>{{ user_time }} {{#tr this}}Local time{{/tr}}</li>
         {{/if}}
 
-        <li>{{ user_type }}</li>
+        {{#if_and is_bot bot_owner }}
+            <li>{{#tr this}}Owner{{/tr}}:
+                <span class="bot-owner-name" data-bot-owner-id='{{ bot_owner.user_id }}'>
+                    {{bot_owner.full_name}}
+                </span>
+            </li>
```

Comment:
@rishig for bots that don't have an owner, do we want to display "Bot" as their user type?  I think that might be clearer than having nothing for them in this spot.

Reply:
(Codewise, if we did that, we could just have it be `if bot_owner` above, and pass a `user_type` of "Bot", and take a bunch of conditionals out of the templates))

Reply:
I didn't realize bots could be ownerless -- is the situation here that they had an owner that is now deactivated?

If a bot does have an owner, is it that we currently put "Member" or "Admin" depending on who owns it?

Reply:
We currently just have that section blank, I think, if it doesn't have an owner?  

I think it's actually a legacy data problem -- we didn't have a "bot owner" column when bots were first created, and we didn't have a good plan for what to do when migrating.  

I'm not sure if there's a way to create a new bot without an owner other than the system bots (e.g. Notification Bot), which don't have owners either (but could potentially say something special like "System Bot" here).

Reply:
"System bot" is great for welcome bot and friends. 
If it's just a legacy data problem, I think optimizing for code simplicity makes sense.

Reply:
OK sounds good. @vrongmeal can you update this?